src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,38 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+
```

Comment:
Consider adding a descriptive header for the Event class? I noticed that you may have missed adding header comments for some other classes as well, so perhaps it would be good to consitently write descriptive header comments for all public classes/methods.